.gitignore

```diff
@@ -1,6 +1,35 @@
 composer.phar
 /vendor/
 
+/.phpunit.cache
+/bootstrap/ssr
+/node_modules
+/public/build
+/public/hot
+/public/storage
+/storage/*.key
+/storage/pail
+/vendor
+.env
+.env.backup
+.env.production
+.phpactor.json
+.phpunit.result.cache
+Homestead.json
+Homestead.yaml
+npm-debug.log
+yarn-error.log
+/auth.json
+/.fleet
+/.idea
+/.nova
+/.vscode
+/.zed
+docker-compose.override.yml
+.DS_Store
+resume.pdf
+Usage.md
+
```

Comment:
Lots of ignores that seem unrelated to the project. In the project gitignore we should strictly only add ignores for files/directories that are found in the project itself, not dependent on the dev's platform or environment. Of the whole list, I think you can keep only `vendor` and `.env`. If you then want to ignore more files, you can add them to your global gitignore file.